nikosnick13.HabitLogger/Program.cs

```diff
@@ -0,0 +1,293 @@
+﻿using Microsoft.Data.Sqlite;
+using System.Data;
+using System.Globalization;
+using static System.Console;
+
+namespace nikosnick13.HabitLogger
+{
+    internal class Program
+    {
+
+
+        static string connectionString = @"Data Source=habit-Traker.db";
+
+        static void Main(string[] args)
+        {
+
+
+            using (var conn = new SqliteConnection(connectionString))
+            {
+                conn.Open();
+
+                var tableCmd = conn.CreateCommand();
+
+                tableCmd.CommandText =
+                    @"CREATE TABLE IF NOT EXISTS  dring_water(
+                        Id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        Date TEXT, 
+                        Quantity  INTEGER
+                    )";
+
+                tableCmd.ExecuteNonQuery();
+
+                conn.Close();
+
+
+
+            }
+
+            UserInput();
+
+            ReadKey();
+
+        }
+
+        //USER INPUT MENY
+        public static void UserInput()
+        {
+            Clear();
+            ForegroundColor = ConsoleColor.Green;
+
+            bool isAppRunning = false;
+
+            while (!isAppRunning)
+            {
+               
+                WriteLine("\n\nMAIN Menu");
+                WriteLine("\nWhat would you like to do?");
+                WriteLine("\n\tType 0 to Close Application.");
+                WriteLine("\tType 1 to Inser a Record ");
+                WriteLine("\tType 2 to View All Records");
+                WriteLine("\tType 3 to Delete a Record");
+                WriteLine("\tType 4 to Update a Record");
+                WriteLine("============================================\n");
+
+                string UserInput = ReadLine();
+
+                switch (UserInput)
+                {
+                    case "0":
+                        WriteLine("\nGoodbye!! \n");
+                        isAppRunning = true;
+                        Environment.Exit(0);
+                        break;
+                    case "1":
+                        Insert();
+                        break;
+                    case "2":
+                        VeiwAll();
+                        break;
+
+                    case "3":
+                        Delete();
+                        break;
+                    case "4":
+                        Updade();
+                        break;
+                    default:
+                        WriteLine("Invalid Command.Please type a number from 0 to 4");
+                        break;
+                }
+
+            }
+        }
+
+        #region Insert
+
+        private static void Insert()
+        {
+            Clear();
+            string date = GetDateInput();
+
+            int quantity = GetNumberInput("\n\nPlaese Insert a number of glasses or another measures of your choice (no decimals is allowed \n\n) ");
```

Comment:
Weird bracket on different line.

`int quantity = GetNumberInput("\n\nPlaese Insert a number of glasses or another measures of your choice (no decimals is allowed \n\n) ");`

`int quantity = GetNumberInput("\n\nPlaese Insert a number of glasses or another measures of your choice (no decimals is allowed) \n\n ");`

Reply:
Also typo:
`PLaese`

`Please`